components/link_tooltip/link_tooltip.tsx

```diff
@@ -45,29 +48,35 @@ export default class LinkTooltip extends React.PureComponent<Props, State> {
         };
     }
 
-    public showTooltip = (e: any): void => {
+    public showTooltip = (e: React.MouseEvent<HTMLSpanElement>): void => {
         //clear the hideTimeout in the case when the cursor is moved from a tooltipContainer child to the link
         window.clearTimeout(this.hideTimeout);
 
         if (!this.state.show) {
-            const $target: JQuery = $(e.target);
-            const target: Element = $target.get(0);
-            const $tooltipContainer: JQuery = $(this.tooltipContainerRef.current);
-            const tooltipContainer: Element = $tooltipContainer.get(0);
+            const target = e.currentTarget;
+            const tooltipContainer = this.tooltipContainerRef.current;
 
             //clear the old this.showTimeout if there is any before overriding
             window.clearTimeout(this.showTimeout);
 
             this.showTimeout = window.setTimeout(() => {
                 this.setState({show: true});
 
-                $tooltipContainer.show(); // eslint-disable-line jquery/no-show
-                $tooltipContainer.children().on('mouseover', () => clearTimeout(this.hideTimeout));
-                $tooltipContainer.children().on('mouseleave', (event: JQueryEventObject) => {
-                    if (event.relatedTarget !== null) {
-                        this.hideTooltip();
+                if (!tooltipContainer) {
+                    return;
+                }
+
+                const addChildEventListeners = (node: Node) => {
+                    node.addEventListener('mouseover', () => clearTimeout(this.hideTimeout));
+                    if (node instanceof HTMLElement) {
```

Comment:
Without this check TS thrown `Property 'relatedTarget' does not exist on type 'Event'.`